[Mini Apps] Add VK mini apps (#6149)
GoogleChrome/web.dev: src/site/content/en/mini-apps/mini-app-alternative-runtime-environments/index.md

  </figcaption>
</figure>

### VK Mini Apps

The Russian social networking platform [VK](https://vk.com/mini-apps) runs a [mini app program](https://vk.com/mini-apps)
that allows developers to [build](https://vk.com/dev/vk_apps_docs) mini apps that tie in deeply with the social network.
VK mini apps work on both VK's platform-specific mobile apps, as well as on the desktop website.
Apart from several of the brand's other platforms like [Mail.ru](https://mail.ru/), VK mini apps are also integrated in the
[Atom browser](https://browser.ru/).

<figure class="w-figure">

>   {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="ALT_TEXT_HERE", width="460", height="948" %}

Author: ```suggestion
  {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="ALT_TEXT_HERE", width="230", height="474" %}
```

>   {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="ALT_TEXT_HERE", width="230", height="474" %}

Author: ```suggestion
  {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="ALT_TEXT_HERE", width="460", height="948" %}
```
```suggestion
  {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="The Все аптеки mini app running in VK", width="230", height="474" %}
```

>   {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="ALT_TEXT_HERE", width="460", height="948" %}

Reviewer: Replace `ALT_TEXT_HERE`

Author: Done.

>   {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="ALT_TEXT_HERE", width="460", height="948" %}

Author: ```suggestion
  {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/tZJgYpdjurBVfhInqWRo.webp", alt="The Все аптеки mini app running in VK.", width="460", height="948" %}
```

  <figcaption class="w-figure">

>     The Vse Apteki mini app running in VK (Source: <a href="https://vk.com/mini-apps">VK</a>).

Author: ```suggestion
    The Все аптеки mini app running in VK (Source: <a href="https://vk.com/mini-apps">VK</a>).
```
